include/SFML/System/Clock.hpp

```diff
@@ -74,11 +75,23 @@ class SFML_SYSTEM_API Clock
     Time restart();
 
 private:
+    using ClockImpl = std::chrono::system_clock;
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Convert clock duration to microseconds
+    ///
+    /// This function acts as a utility for converting clock
+    /// duration type instance into sf::Time
+    ///
+    /// \return Time in microseconds
+    ///
+    ////////////////////////////////////////////////////////////
+    static Time toMicroseconds(ClockImpl::duration duration);
```

Comment:
`sf::Time` is not microseconds, but an opaque type which is convertible to microsecond duration. Maybe name the method `durationToTime()` or so?